prettier.config.js

```diff
@@ -1,4 +1,12 @@
 module.exports = {
   trailingComma: 'es5',
   singleQuote: true,
+  overrides: [
+    {
+      files: '*.json',
+      options: {
+        printWidth: 400,
```

Comment:
Not sure if you guys want that smaller, but I added 400 just to avoid wrappings in JSON files.. up to you